src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Public.Data/DataSetVersionService.cs

```diff
@@ -325,7 +354,7 @@ private async Task<ReleaseFile> GetReleaseFile(
     {
         return await contentDbContext.ReleaseFiles
             .AsNoTracking()
-            .Where(rf => rf.Id == dataSetVersion.ReleaseFileId)
+            .Where(rf => rf.Id == dataSetVersion.Release.ReleaseFileId)
```

Comment:
I would have expected this to blow up because we didn't do a `.Include(dsv => dsv.Release)` when grabbing the `DataSetVersion` which is passed into this method

Reply:
That's going to be covered by the answer to the previous comment as well. The `DataSetVersion` passed into this method will have the owned `Release` type included by default.